ci/docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+version: "3.9"
+
+services:
+  node1:
+    image: ghcr.io/xline-kv/xline:latest
+    networks:
+      xline_network:
+        ipv4_address: 172.22.0.2
+    volumes:
+      - ./fixtures:/mnt
+    ports:
+      - "2379:2379"
+    environment:
+      RUST_LOG: curp=debug,xline=debug
+    command: >
+      xline
```

Comment:
Please add `--client-listen-urls` and `--peer-listen-urls` in the xline startup command.